proto/p4/config/p4info.proto

```diff
@@ -21,14 +21,12 @@ import "google/protobuf/any.proto";
 package p4.config;
 
 message P4Info {
-  repeated Table tables = 1;
-  repeated Action actions = 2;
-  repeated ActionProfile action_profiles = 3;
-  repeated Counter counters = 4;
-  repeated Meter meters = 5;
-
-  // for extern?
-  repeated google.protobuf.Any externs = 100;
+  repeated Extern externs = 1;
```

Comment:
I believe this was @wmohsin 's suggestion. Since we are likely to add native support for standard externs (e.g. registers, qos?) to this list, it may make more sense to have externs at the beginning to be able to grow the list as much as we need to. That being said, we are unlikely to reach 100 standard objects. I don't have a strong preference.